src/app/tests/suites/manualTests.json

```diff
@@ -0,0 +1,423 @@
+{
+    "DeviceDiscovery": [
```

Comment:
The .js files have some comments about why things are manual (and should have more; ideally everything that is manual would have an explanation of _why_ it's manual).  We should support that in this format.